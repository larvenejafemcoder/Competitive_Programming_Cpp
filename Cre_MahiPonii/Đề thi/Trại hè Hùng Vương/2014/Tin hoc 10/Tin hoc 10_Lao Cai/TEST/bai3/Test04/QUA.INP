20 25
23 10 7 2 2 19 11 1 24 9 2 22 6 21 5 8 0 0 11 12 19 10 24 11 15 
6 22 9 18 0 8 1 14 13 0 6 2 3 4 7 9 0 4 11 14 9 17 17 8 16 
15 23 18 14 24 4 19 23 11 14 2 13 6 9 6 17 24 16 16 2 11 1 9 5 19 
22 14 5 12 16 7 1 11 15 17 0 4 4 15 7 20 11 0 22 10 23 24 23 23 10 
13 12 4 15 22 24 0 24 7 12 24 8 21 5 13 13 12 10 11 5 15 14 8 20 19 
23 3 8 4 15 16 19 10 13 24 23 6 9 22 12 10 0 13 21 4 18 10 13 4 7 
2 5 3 8 12 11 0 23 7 24 18 18 8 20 15 2 11 7 14 22 3 9 5 3 19 
1 15 16 14 8 9 21 17 7 20 18 4 24 5 6 16 20 2 7 4 19 13 7 16 12 
3 3 10 9 0 12 14 23 9 23 8 23 4 21 24 12 18 24 10 1 24 3 6 4 15 
5 13 21 19 16 9 7 9 8 19 16 11 12 14 21 9 6 2 4 13 2 1 15 10 0 
8 1 2 7 12 8 15 19 18 0 4 14 21 23 13 2 11 0 19 24 23 9 8 21 23 
18 23 23 14 13 23 5 23 23 3 0 23 1 4 13 2 22 16 15 10 9 2 12 3 7 
23 4 22 24 6 18 5 1 13 17 17 0 17 23 15 9 1 16 22 1 13 21 2 9 5 
22 19 10 15 16 17 22 12 15 7 17 11 19 7 13 0 10 8 15 4 1 0 7 6 6 
2 20 5 13 8 16 0 10 20 0 19 6 13 17 13 1 10 6 15 18 1 21 20 7 24 
11 24 13 23 20 17 24 2 21 9 18 24 24 12 4 17 14 14 10 12 10 2 11 16 18 
24 16 5 21 22 6 7 21 21 10 20 23 19 5 3 19 12 18 0 11 15 2 23 5 23 
2 1 11 13 10 23 13 24 14 9 13 8 3 21 7 22 7 23 24 9 0 18 5 0 5 
9 12 7 9 8 18 8 4 2 10 18 10 8 8 7 20 22 3 17 11 10 15 2 3 3 
24 15 12 24 22 18 22 18 7 2 12 21 13 5 17 5 19 5 20 2 1 18 23 23 19 
