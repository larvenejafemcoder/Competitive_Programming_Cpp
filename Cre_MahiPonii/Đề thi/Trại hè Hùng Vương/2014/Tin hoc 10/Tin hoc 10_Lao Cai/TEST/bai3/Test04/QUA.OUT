498
  17  16  12  13  14  12  14  17  16  13   5  12  16  11   9   8   8   9  18   7  11   5   4   4   3